generator_process/actions/huggingface_hub.py

```diff
@@ -109,12 +124,12 @@ def hf_snapshot_download(
     )
     from huggingface_hub.file_download import REGEX_COMMIT_HASH, repo_folder_name, hf_hub_url, _request_wrapper, hf_raise_for_status, logger, cached_download, build_hf_headers, get_hf_file_metadata, _cache_commit_hash_for_specific_revision, OfflineModeIsEnabled, _create_relative_symlink
     from huggingface_hub.hf_api import HfApi
-    from huggingface_hub.utils import filter_repo_objects, validate_hf_hub_args, tqdm, logging, EntryNotFoundError, LocalEntryNotFoundError
+    from huggingface_hub.utils import filter_repo_objects, validate_hf_hub_args, tqdm, logging, EntryNotFoundError, LocalEntryNotFoundError, RevisionNotFoundError
 
     from diffusers import StableDiffusionPipeline
     from diffusers.utils import DIFFUSERS_CACHE, WEIGHTS_NAME, CONFIG_NAME, ONNX_WEIGHTS_NAME
     from diffusers.schedulers.scheduling_utils import SCHEDULER_CONFIG_NAME
-    from diffusers.hub_utils import http_user_agent
+    from diffusers.utils.hub_utils import http_user_agent
     config_dict = StableDiffusionPipeline.get_config_dict(
```

Comment:
FutureWarning:  The function get_config_dict is deprecated. Please use <class '...StableDiffusionPipeline'>.load_config instead. This function will be removed in version v1.0.0
```suggestion
    config_dict = StableDiffusionPipeline.load_config(
```